articles/build-a-restful-api-with-adonisjs/index.md

```diff
@@ -0,0 +1,419 @@
+### Build A RESTful API With AdonisJS
+
+### Outline
+Introduction
+Setting up AdonisJS
+Creating Databases
+Setting up Authorization and Authentication
+Creating Models
+Creating Controllers
+Creating Endpoint Routes
+Testing the Forum API
+Conclusion
+
+### Introduction
+In today’s ever-changing and rapid world of frameworks including frontend and backend frameworks, learning and building RESTful APIs to bridge the gap between clients and data is a demanding skill for any software engineer.
+
+"A RESTful API is an application program interface (API) that uses HTTP requests exchange data between parties using endpoints (routes) with either GET, PUT, POST and DELETE as the HTTP requests." from [TowardDataScience](https://towardsdatascience.com)
```

Comment:
... HTTP requests `to` exchange data between parties using endpoints

Reply:
done